bin/rad-project

```diff
@@ -13,7 +13,7 @@
 (import prelude/cmd-parsing :unqualified)
 (import prelude/io :as 'io)
 (import prelude/io-utils
-  '[set-git-config get-git-config process-git-with-exit!]
+  '[set-git-config! get-git-config! process-git-with-exit!]
```

Comment:
The changes in this PR surfaced these two typos. I'm a bit confused how the code in `rad-project` was able to run with the erroneous imports.

Reply:
Ha. Turns out prelude already includes all of `io-utils`, which is why it was working. I can remove that import from prelude (it shouldn't really be there) in a separate PR.

Reply:
I see!